src/time.cr

```diff
@@ -524,6 +524,26 @@ struct Time
     utc(seconds: seconds, nanoseconds: nanoseconds.to_i)
   end
 
+  # Creates a new `Time` instance representing the current time from the
+  # system clock observed in *location* (defaults to local time zone).
+  #
+  # DEPRECATED: `Time.now` is deprecated, use `Time.local` or `Time.utc` instead.
```

Comment:
We are ready for a rebase and remove the deprecated comment here and in the other method.

Reply:
Forgot to push 🕺